hmiServerDev.sh

```diff
@@ -1,9 +1,39 @@
+#!/bin/bash
```

Comment:
Nitpick: might want to consider `#!/usr/bin/env bash` shebang

https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash